partcad/src/partcad/part_factory_scad.py

```
@@ -74,16 +81,16 @@ async def instantiate(self, part):
-            try:
-                shape = b3d.Mesher().read(stl_path)[0].wrapped
-            except:
+            with telemetry.tracer.start_as_current_span("*PartFactoryScad.instantiate.{build123d.import_stl}"):
                 try:
-                    # First, make sure it's not the known problem in Mesher
-                    shape = b3d.import_stl(stl_path).wrapped
-                except Exception as e:
-                    part.error("%s: %s" % (part.name, e))
-                    return None
+                    shape = b3d.Mesher().read(stl_path)[0].wrapped
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)